modules/kubernetes-addons/main.tf

```diff
@@ -588,3 +588,5 @@ module "local_volume_provisioner" {
   helm_config   = var.local_volume_provisioner_helm_config
   addon_context = local.addon_context
 }
+
+# whitespace noise
```

Comment:
this is just to ensure that CI picks up a diff and runs checks - will be removed